drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -4635,7 +4635,6 @@ Transform2D RasterizerStorageGLES3::multimesh_instance_get_transform_2d(RID p_mu
 }
 
 Color RasterizerStorageGLES3::multimesh_instance_get_color(RID p_multimesh, int p_index) const {
-
```

Comment:
It would be better not to remove this line, as it marks this file as modified by this commit even though there is no functional changes.

Reply:
Oh, I did not change that file, it happened accidentally somehow. :)

Reply:
If you can undo the change next time you amend the commit, that would be great :)
But it's not a big deal either if it stays this way.